src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -80,11 +81,26 @@ public virtual void SetupChannel(IEnumerable<string> sources)
 
                 var portNumber = this.RequestSender.InitializeCommunication();
                 var processId = this.processHelper.GetCurrentProcessId();
-                var connectionInfo = new TestRunnerConnectionInfo { Port = portNumber, RunnerProcessId = processId, LogFile = this.GetTimestampedLogFile(EqtTrace.LogFile) };
+
+                var logFileName = this.GetTimestampedLogFile(EqtTrace.LogFile);
```

Comment:
Do we expect a consumer of EqtTrace to check for permissions? E.g vstest.console may still fail today if there is no write access to current directory? Or file system has no space etc?